CONTRIBUTING.md

```diff
@@ -2,78 +2,97 @@
 
 Hi! We're really excited that you are interested in contributing to ACPaaS UI. Before submitting your contribution though, please make sure to take a moment and read through the following guidelines.
 
-- [Code of Conduct](CODE_OF_CONDUCT.md)
-- [Issue Reporting Guidelines](#issue-reporting-guidelines)
-- [Pull Request Guidelines](#pull-request-guidelines)
-- [Development Setup](#development-setup)
-- [Project Structure](#project-structure)
+- [ACPaaS UI Contributing Guide](#acpaas-ui-contributing-guide)
+  - [Issue Reporting Guidelines](#issue-reporting-guidelines)
+  - [Pull Request Guidelines](#pull-request-guidelines)
+  - [Development Setup](#development-setup)
+  - [Project Structure](#project-structure)
+  - [Releases](#releases)
 
 ## Issue Reporting Guidelines
 
-- Always use [GitHub Issues][github-issues] to create new issues.
+- Use the [#acpaas-ui channel](https://dgpls.slack.com/messages/C4M60PQJF) on slack to ask questions about using ACPaaS UI.
```

Comment:
Now I think again about using slack as communication channel. The digipolis Slack isn't accessible for users with no digipolis.be mailbox.

Reply:
Hmm, I forgot about that one. TBD what strategy to use. Options: open up dgpls slack, make a separate dgpls-open slack space, make a separate acpaas-ui slack space, or make a gitter space.

Reply:
Was discussed internally with the enterprise architects. The consensus was to create a separate slack space for all things open at digipolis (open source, open data, community, ...). The acpaas-ui channels will move to this new slack space.

Reply:
Will make it a separate issue to update the slack url's to an openly accessible slack space